imap_processing/glows/utils/constants.py

```diff
@@ -72,3 +76,12 @@ class DirectEvent:
     timestamp: TimeTuple
     impulse_length: int
     multi_event: bool = False
+
+    def to_array(self):
```

Comment:
nit, but I'd call this `to_list()` to be explicit that you aren't a numpy array just yet.